Gems/ROS2/Code/Source/VehicleDynamics/ModelLimits/SkidSteeringModelLimits.cpp

```diff
@@ -61,6 +61,15 @@ namespace ROS2::VehicleDynamics
         }
     }
 
+    SkidSteeringModelLimits::SkidSteeringModelLimits(
+        const float linearLimit, const float angularLimit, const float linearAcceleration, const float angularAcceleration)
+    {
+        m_linearLimit = AZStd::clamp(linearLimit, 0.0f, 100.0f);
+        m_angularLimit = AZStd::clamp(angularLimit, 0.0f, 10.0f);
+        m_linearAcceleration = AZStd::clamp(linearAcceleration, 0.0f, 100.0f);
+        m_angularAcceleration = AZStd::clamp(angularAcceleration, 0.0f, 100.0f);
```

Comment:
This seems to do the same as SetAngularAccelerationLimit member function. Call it here, or remove the function if it is not meant to be used in isolation (limits are set together each time).

Reply:
I removed the constructor as AngularAccelerationLimit is the only limit I set.